src/Apps/ArtAdvisor/01-Spike/server.ts

```diff
@@ -0,0 +1,15 @@
+import express from "express"
+
+export const router = express.Router()
+
+router.get("/", (req, res) => {
+  res.send("Hello root under experiment 1")
+})
+
+router.get("/a", (req, res) => {
+  res.send("Hello experiment 1a")
+})
+
+router.get("/b", (req, res) => {
+  res.send("Hello experiment 1b")
+})
```

Comment:
Do we need to change the `fetch` request in the related `App.tsx` ? Maybe doesn't matter, as `02-Markdown` is just an improved version?

Reply:
01 still has the placeholder routing. 

I focused on 03 for the demo, and then was working backwards: wired up 02, but then I wasn't sure which Quantum backend 01 was paired with.

Reply:
Got it. I think that is fine. IIRC it was originally for the `03-express-streaming` app, but it was changed to test some of the other approaches. Don't think we will need to go back to this, so probably fine to leave as is.

Reply:
I timeboxed trying to pull that quantum endpoint into here, but it wasn't working. Since we agreed we were fine if demo 01 wasn't wired up, I marked it as such on the landing page.